api-report/sequence.api.md

```diff
@@ -104,7 +105,7 @@ export interface IMapMessageLocalMetadata {
 
 // @public
 export class Interval implements ISerializableInterval {
-    constructor(start: number, end: number, props?: PropertySet);
+    constructor(start: number, end: number, props?: PropertySet, stickiness?: IntervalStickiness | undefined);
```

Comment:
probably stick with optional

Reply:
I think the API generator adds the `| undefined`? This is the declaration of the constructor:

![image](https://user-images.githubusercontent.com/39542938/235746602-1da355d7-433f-4402-b26d-640f2e887fcc.png)

It does look like an issue on my end as this is the only type I can find that's both optional and `| undefined`, but I don't see anything weird with how I've declared the type.